### ISSUE TITLE
False positive in FileName cop

### ISSUE THREAD

**Issue by sferik** [rubocop/rubocop#890]
RuboCop version 0.19.0 is reporting the following `FileName` error:

```
Use snake_case for source file names.
```

in this [code](https://github.com/intridea/omniauth-oauth2/blob/48320bcd92cf6def8ea21a6bde67aabcfa0c09d6/lib/omniauth-oauth2.rb#L2):

``` ruby
require 'omniauth/strategies/oauth2'
```

Perhaps this is because the file name ends with a number?
